<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>410</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>451</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Log Details</string>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>431</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(235, 235, 235);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="currentVersionLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Version :</string>
    </property>
   </widget>
   <widget class="QLabel" name="latestVersionLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Latest Version :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadBtn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>451</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Downloading</string>
    </property>
    <widget class="QLabel" name="fileNameLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>401</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>File Name :</string>
     </property>
    </widget>
    <widget class="QLabel" name="fileSizeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>401</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>File Size :</string>
     </property>
    </widget>
    <widget class="QLabel" name="alreadDownloadLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>401</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Already Download :</string>
     </property>
    </widget>
    <widget class="QLabel" name="speedLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>401</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed :</string>
     </property>
    </widget>
    <widget class="QLabel" name="timeLeftLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>401</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Time Left :</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>431</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>509</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
